daemon/cmld.c

```diff
@@ -963,7 +1004,19 @@ cmld_tune_network(const char *host_addr, uint32_t host_subnet, const char *host_
 	/* configure loopback interface of root network namespace */
 	network_setup_loopback();
 
-	cmld_netif_phys_list = network_get_physical_interfaces_new();
+	list_t *ifis = network_get_physical_interfaces_new();
+
+	for (list_t *l = ifis; l; l = l->next) {
+		char *ifi_new = rename_ifi(l->data);
+
+		if (ifi_new) {
+			DEBUG("Got physical network interface %s", ifi_new);
+			cmld_netif_phys_list = list_append(cmld_netif_phys_list, ifi_new);
+		} else {
+			ERROR("Failed to rename ifi, using as %s", (char *)l->data);
+			cmld_netif_phys_list = list_append(cmld_netif_phys_list, ifi_new);
```

Comment:
Is this line really intended?

Reply:
yes, because even if the renaming fails (should only happen in rare corner cases) the cmld should manage this interface

Reply:
done